data/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="default_fallback_handler">DefaultFallbackHandler</string>
+    <string name="unknown_fallback_handler">Unknown</string>
+    <string name="empty_string"></string>
```

Comment:
Does it really make sense to have an empty string in a translatable file? I would at least mark it as not translatable.

Also it feels weird to have string resources in the data package imo as these are more "app" (UI) specific

Reply:
You're right. Those string resources do not have to be there.